src/res/translation/translation_de_DE.ts

```diff
@@ -3608,12 +3608,12 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
         <location filename="../../serverdlg.cpp" line="125"/>
         <source>Recording has been switched off by the UI checkbox</source>
-        <translation type="unfinished"></translation>
+        <translation>Die Aufnahme wurde durch eine Auswahl in der Benutzeroberfläche beendet</translation>
```

Comment:
```suggestion
        <translation>Die Aufnahme wurde über die Checkbox in der Benutzeroberfläche ausgeschaltet</translation>
```
I think this is about disabling recording altogether, so it's just just stopped, it's disabled, isn't it?

Reply:
Not sure. Probably yes. The SIGUSR command is mentioned somewhere else